cmd/jaeger/integration/datareceivers/kafka.go

```diff
@@ -0,0 +1,53 @@
+package datareceivers
```

Comment:
> cmd/jaeger/integration/datareceivers/kafka.go

can we keep this all under $root/internal/cit/?

Reply:
Sure!